src/local-history/local-history-manager.ts

```diff
@@ -198,11 +202,18 @@ export class LocalHistoryManager {
      * @param uri: the target uri for the new file.
      * @param content: the data to be stored in the file
      */
-    private writeFile(uri: string, content: string): Promise<string> {
+    private writeFile(uri: string, content: string, changePermission: boolean): Promise<string> {
         const writeStream = fs.createWriteStream(uri);
         return new Promise((resolve) => {
             writeStream.on('open', () => writeStream.write(content));
             writeStream.on('end', () => resolve());
+
+            console.log('Cannot change permission if I delete this console.log...');
```

Comment:
For some reason, I can't change the file permission to `read-only` if I remove this line